src/main/webapp/app/exercises/programming/manage/update/update-components/programming-exercise-language.component.html

```diff
@@ -172,6 +172,23 @@
             <jhi-help-icon placement="auto" text="artemisApp.programmingExercise.recordTestwiseCoverageTooltip" />
         </div>
     }
+    <!-- Setting whether the solution repository should be checked out is only supported for OCAML and Haskell exercises and is only available for new exercises -->
```

Comment:
```suggestion
```
I think we can remove this. The selection functionality here is independent of that and in the section setting the flag, it's mentioned.